src/OpenColorIO/BitDepthUtils.cpp

```diff
@@ -69,6 +69,27 @@ OCIO_NAMESPACE_ENTER
         }
     }
 
+    BitDepth GetBitdepthFromMaxValue(unsigned maxValue)

```

Comment:
I'll propose adding the following comment:

// For formats that do not explicitly identify the intended bit-depth scaling, we must infer it based on the LUT values.  However LUTs sometimes contain values that extend outside the nominal ranges.  For example, a LUT that started out in a floating point format with values going up to 1.09 may get converted to another format that uses 10-bit values and those extend up to 1.09 * 1023 = 1115.  In this case we want the "auto detection" to return 10-bit rather than 12-bit.  Hence rather than using breakpoints of 1024, 2048, 4096, etc., we use breakpoints that are midway between in order to better handle LUTs with occasional over-range values.

Reply:
That clarifies the usage nicely

Reply:
And the new implementation is super clear. Thanks!